pipeline/02-assess.R

```diff
@@ -362,15 +363,27 @@ sales_data_ratio_study <- sales_data %>%
 sales_data_two_most_recent <- sales_data %>%
   distinct(
     meta_pin, meta_year,
-    meta_sale_price, meta_sale_date, meta_sale_document_num
+    meta_sale_price, meta_sale_date, meta_sale_document_num,
+    sv_outlier_type
+  ) %>%
+  rename(meta_sale_outlier_type = sv_outlier_type) %>%
+  mutate(
+    meta_sale_outlier_type = ifelse(
+      meta_sale_outlier_type == "Not outlier", NA, meta_sale_outlier_type
+    )
```

Comment:
I would make a note here that we're including outliers. Note also to recrawl once this is merged and run since it will make new output columns in `assessment_pin`.

Reply:
Done as of e02cdd29b4cc03e629f9f0abeff53a51fccb632b!